src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/ExtensionManagement/ExtensionManagementCommand.cs

```diff
@@ -64,7 +88,7 @@ public AddExtensionCommand()
             {
                 AddHandler<AddExtensionAppCommand>();
                 AddArgument(new Argument<string>("name", LocalizedStrings.ExtensionManagementName));
-                AddOption(new Option<string>(new[] { "--source" }, () => DefaultSource, LocalizedStrings.ExtensionManagementSource));
+                AddOption(new Option<string>(new[] { "--source" }, LocalizedStrings.ExtensionManagementSource));
```

Comment:
Default source is not needed anymore?

Reply:
It's not a constant any more, but a setting brought in via appsettings.json